pkg/stringutils/split.go

```diff
@@ -58,3 +58,17 @@ func GetAfterLast(str, sep string) string {
 	}
 	return p2
 }
+
+// GetBetween gets the string between the two passed strings, otherwise it returns the empty string
+func GetBetween(str, start, end string) string {
+	startIdx := strings.Index(str, start)
+	if startIdx == -1 || startIdx == len(str)-len(start) {
+		return ""
+	}
+	offset := startIdx + len(start)
+	endIdx := strings.Index(str[startIdx+len(start):], end)
```

Comment:
nit: `endIdx := strings.Index(str[offset:], end)`

Reply:
Thanks, had refactored to offset :)